src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -399,6 +403,15 @@ class Microsoft::Terminal::Core::Terminal final :
     std::optional<KeyEventCodes> _lastKeyEventCodes;
 
     std::vector<Microsoft::Console::VirtualTerminal::DispatchTypes::ScrollMark> _scrollMarks;
+    enum PromptState : uint32_t
+    {
+        None = 0,
+        Prompt = 1,
+        Command = 2,
+        Output = 3

```

Comment:
nit: You can drop the explicit numbers if you'd like. The default behavior of enums is identical. Also I believe you meant to write `enum class` right?